packages/react/src/components/ThemeSettings/components/index.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export { ThemeSettings } from './ThemeSettings';
+export { ThemeSwitcher } from './ThemeSwitcher';
+export { ThemeSetDropdown } from './ThemeSetDropdown';
+export { ThemeMenuCompliment } from './ThemeMenuCompliment';
```

Comment:
```suggestion
export { ThemeMenuComplement } from './ThemeMenuComplement';
```

Reply:
Thanks @elycheea I'd blame the lack of cspell, but it does not spot homophones.